.gitattributes

```diff
@@ -1,3 +1,4 @@
+ios/Exponent.xcodeproj/project.pbxproj merge=mergepbx -text linguist-generated
```

Comment:
Hmm, still, what about other project.pbxproj? Especially from Pods folder, in the client and bare-expo as well.